### DISCUSSION TITLE
Cannot Install on Matebook X Pro 2019 - Intel i7 8th Gen (Error 0x80370102)

### DISCUSSION THREAD

**Discussion by gatotj** [WhitewaterFoundry/Pengwin#765] [Q&A]
I can't install Pengwin on WSL2 of Windows 11 (purchased and downloaded through Windows Store), giving the infamous 0x80370102 error during unpacking. It asked to enable Virtual Machine Platform and enabling virtualization in BIOS.

I have enabled VM Platform, both using Turn Windows Feature On Off and dism method. Using dism, Powershell completed normally, giving us info that VM Platform have been enabled.

I cannot really change virtualization in BIOS (UEFI) in this laptop, because no option for it within Insyde's BIOS. However, Huawei said if the option is not given, it is automatically enabled. I cross-checked this using Task Manager (on the CPU section) that it is indeed enabled. I'm using BIOS v1.37.

I have also browse the internet, applying recommendation to similar problem (however pre-2020), which is enabling HyperV. However, this doesn't help.

Do you guys have similar experience? How did you solve it?

Many thanks.
![Pengwin Problem Snapshots](https://user-images.githubusercontent.com/12730658/223034111-0a04676b-7dab-4939-8e3d-0e5048d2d7c7.png)

**Comment by crramirez** [comment]
Hello,

Here is a similar discussion https://github.com/WhitewaterFoundry/Fedora-Remix-for-WSL/issues/78#issuecomment-633363296

But I think that your case is different. With Virtual Machine Platform is enough. There is a flag in the boot partition that must be turned on. Windows turn it on when you install the Virtual Machine Platform but sometimes is stuck. I'll look for it later.

If you cannot enable the virtualization, depending on your use case you can go for WSL1 that doesn't requires it.
